app/controllers/taxi_requests_controller.rb

```diff
@@ -11,4 +11,27 @@ def index
 
     json_success taxi_requests.map { |request| TaxiRequestSerializer.new(request).serialized_json }
   end
+
+  def create
+    params = create_param
+    pending_request = TaxiRequest.find_by(status: TaxiRequest.statuses[:pending])
+    raise Exceptions::Forbidden, '승객만 배차 요청할 수 있습니다' unless current_user.passenger?
+    raise Exceptions::Conflict, '아직 대기중인 배차 요청이 있습니다' unless pending_request.nil?
+
+    request = TaxiRequest.create!(
+      passenger_id: params[:passenger_id],
+      driver_id: 0,
+      address: params[:address],
+      status: TaxiRequest.statuses[:pending]
+    )
+
+    json_create_success TaxiRequestSerializer.new(request).serialized_json
+  end
+
+  private
+
+  def create_param
```

Comment:
네이밍을 복수하시면 좋을 것 같습니다.

```suggestion
  def create_params
```

Reply:
넵 해당 부분 반영하였습니다 :)